### ISSUE TITLE
/var/lib/docker/overlay2 accumulates files over time

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#16381]
**Describe the bug**
The directory `/var/lib/docker/overlay2` accumulates files over time.

**To Reproduce**
1. Go to a long-running BigBlueButton 2.5/2.6 server
2. Run the command `ls -alt /var/lib/docker/overlay2`

**Expected behavior**
This directory doesn't have old files.

**Actual behavior**
Since the libreoffice document is ephemeral, didn't expect to see files in this directory over time.  It's possible to clear the files out with `docker system prune -a`.

**Additional Information**
Looking on our dev26, we could see the files accumulating

```
# ls -alt /var/lib/docker/overlay2
total 404
drwx--x--- 99 root root 12288 Dec 30 21:08 .
drwx------  2 root root  4096 Dec 30 21:08 l
drwx--x---  5 root root  4096 Dec 20 14:05 e97fd1c451ac45d67e71f697c00cfd0156ebee3324fb594f56435cfce259c03c
drwx--x---  5 root root  4096 Dec 20 14:05 f556077166580578ee6b65e26d2da2112023b29f33d437fdd510c4a7f8b7d50a
drwx--x---  5 root root  4096 Dec 20 14:05 482bc2e06712acbac6c1be37a9d518f0ac538c405e362cc2d6f0c6835165d91c
drwx--x---  5 root root  4096 Dec 20 14:05 54407b1623c1e9cb4c0fd1f5403ac1ecae9d7a7fc4c4e92f05a1068df9253628
drwx--x---  5 root root  4096 Dec 20 14:05 564c16d8e17f83197b1e03f63a91e69c309d55eabc72b5c5ef588229e4ab9a83
drwx--x---  5 root root  4096 Dec 20 14:05 88df415dfbf98dd81a9c84171dcf9e23a9b7fd6320f2c21ee2edcb0d6c37c896
drwx--x---  5 root root  4096 Dec 20 14:05 b83d41476dc02a72108c9de623f31037ed06d360e1d9e3d19c9dc50a6bd57da9
drwx--x---  5 root root  4096 Dec 20 14:05 dc28a49036a5a3ecb985f7a4fadeeffb811d415547ed1d2ff47c8348be217c99
drwx--x---  5 root root  4096 Dec 20 14:05 0687fc1f7b022bacd4555a47182a80941aa706593da6d2f720df244c8c4e9cd4
drwx--x--- 13 root root  4096 Dec 20 14:05 ..
drwx--x---  4 root root  4096 Sep 30 18:22 042e674daede6107fa7e42c094e6bef32a4c7805f1424b8d994a854f4e41bf8d
drwx--x---  4 root root  4096 Sep 29 17:46 8b2c09209ae886444060ae622f1419bddddbc00e4c593b629f7121fb0ac133df
drwx--x---  3 root root  4096 Sep 29 17:46 8ed0985d6a84fc657087324d794436796512d98b9fe1fef2f5161ba28b287d02
drwx--x---  4 root root  4096 Sep 26 15:51 5b89340517d7e708cc18952fa4bdfdeede969fc34ee7f05236b39ae33909eaff
drwx--x---  4 root root  4096 Aug 24 17:24 ab5f4af33a3033c8339feaba54910737679062ae6a43f9051ddda39872191ec6
drwx--x---  4 root root  4096 Aug 24 15:22 a350c34ad25becca8e0a0ad6370e156aef06cc92b37119f152273bf649b3c91a
drwx--x---  4 root root  4096 Aug 24 15:22 5615b8600c278a0d97a94996cbe568f122f26523954c5be05d39b24fdee0dba9
drwx--x---  4 root root  4096 Aug 24 15:22 e18d77031e77cd28293024296b83b26711d831b67fe3d549a99afc215ff29059
drwx--x---  4 root root  4096 Aug 24 15:22 659589b46620e7b91e6b81e667a2d1d33cfe6931fa1de63a490a636304d8a2a7
drwx--x---  3 root root  4096 Aug 24 15:22 6a52a40a265fc41b242bc35ea9f4723f3094d1cc391a167df8013cddde1e0a35
drwx--x---  4 root root  4096 Jul 27 23:21 88df415dfbf98dd81a9c84171dcf9e23a9b7fd6320f2c21ee2edcb0d6c37c896-init
drwx--x---  4 root root  4096 Jul 27 23:21 dc28a49036a5a3ecb985f7a4fadeeffb811d415547ed1d2ff47c8348be217c99-init
drwx--x---  4 root root  4096 Jul 27 23:21 e97fd1c451ac45d67e71f697c00cfd0156ebee3324fb594f56435cfce259c03c-init
drwx--x---  4 root root  4096 Jul 27 23:21 0687fc1f7b022bacd4555a47182a80941aa706593da6d2f720df244c8c4e9cd4-init
drwx--x---  4 root root  4096 Jul 27 23:21 54407b1623c1e9cb4c0fd1f5403ac1ecae9d7a7fc4c4e92f05a1068df9253628-init
drwx--x---  4 root root  4096 Jul 27 23:21 482bc2e06712acbac6c1be37a9d518f0ac538c405e362cc2d6f0c6835165d91c-init
drwx--x---  4 root root  4096 Jul 27 23:21 564c16d8e17f83197b1e03f63a91e69c309d55eabc72b5c5ef588229e4ab9a83-init
drwx--x---  3 root root  4096 Jul 27 23:21 2c7aae20bdddf389245f241f331b7db42dbc8582b1b9384c741c2877f386c008
drwx--x---  4 root root  4096 Jul 27 23:21 b1eb87c0307ed34e70646d7a589baef54eebb43c26691eddd9c3dea0df3be5dc
drwx--x---  4 root root  4096 Jul 27 23:21 eb94f4e4562d826b8b7e3b0db11480724589450154d3b9e9c2a4015fcd372c95
drwx--x---  3 root root  4096 Jul 27 23:21 2d0b3e185b7ca4ec49b83c8307dac77c2410ef27c1d86af1cb0417eb36505e3e
drwx--x---  4 root root  4096 Jul 27 23:21 9a33b1aa0b2aac89858d95d1f746e786b01a1fc567993fd7f04e5ebdc0576b27
drwx--x---  4 root root  4096 Jul 27 23:21 7295d4089db62c9ae30121d24fa8fc2cd5616ddb8a174658898c2578880d33bd
drwx--x---  4 root root  4096 Jul 27 23:21 21757e4bb43c98f7a5e754b1540592fc14abf46d14a55ebd61feeda47bf665f6
drwx--x---  4 root root  4096 Jul 27 23:21 6564ac5b55441360780b9fc3add7382696cd20bdf2a57fee3f2ccc4a0f7fb59f
drwx--x---  3 root root  4096 Jul 27 23:21 0ba02fb35f13617cad53ea1ba584b4518732f65e0e5610049ae3ff3335fdac42
drwx--x---  3 root root  4096 Jul 27 23:21 0de1b4acd62e0802a499e1c7dda65e8245ebefd12e05ea8b28d9b63c897dac49
drwx--x---  4 root root  4096 Jul 27 23:21 afda1e7343a1be91ec01ba898eb6c01db6c05d484e7e009889b3a366bb2d72f0
drwx--x---  3 root root  4096 Jul 27 23:21 2f96c38f2f36cfd2d33ac673d289bae29130d5743cd3a33bbb48ada4e88ea9f4
drwx--x---  4 root root  4096 Jul 27 23:21 841aa198347901426fad6fb575bbd02a4ecc334315f56c9a30a49ad21ad8892c
drwx--x---  3 root root  4096 Jul 27 23:21 052a1b77869a8b33825e6139693d1a4471b64eb1bf10309ab6a951ed10f2840d
drwx--x---  4 root root  4096 Jul 27 23:21 e48f298acca0cfb483d7b59aaf7efd75a6b1be16f9757b0548fddde55788ea94
drwx--x---  3 root root  4096 Jul 27 23:21 8f1fd119c078cc6c1524da3b4b1ed6f6c5e909ed16478064338fd0ab0ee0fa62
drwx--x---  4 root root  4096 Jun 22  2022 b83d41476dc02a72108c9de623f31037ed06d360e1d9e3d19c9dc50a6bd57da9-init
drwx--x---  4 root root  4096 Jun 22  2022 f556077166580578ee6b65e26d2da2112023b29f33d437fdd510c4a7f8b7d50a-init
drwx--x---  4 root root  4096 Jun 22  2022 4c99e6a14e34ae0b0ab4425eb92a4eb170aa7943e124de646af3bd94807a7b1f
drwx--x---  4 root root  4096 Jun 22  2022 7b504c6634d90ad9c83ed2232f6cf72a97826db1327a92ee2ec76b32ee2162e1
drwx--x---  4 root root  4096 Jun 22  2022 f389470a27d148eee50f02545afce3f2610b8271ca462e8a3c4bb8bb414005aa
drwx--x---  4 root root  4096 Jun 22  2022 e59fb713ad2c993336565710e046093af147acb6bc136cc6f561749c965bc829
drwx--x---  4 root root  4096 Jun 22  2022 945b55e5031ce0a534b27714fca42d4499cec0798ef200a7822d59d3f26f682e
drwx--x---  4 root root  4096 Jun 22  2022 7535d39b0c4b5cd0477acb590194682cf056556db283d7efd35470a81ca1699c
drwx--x---  4 root root  4096 Jun 22  2022 7decef6a20caaeaa4617584f99b5b2e521d8ccec401f3c3c3b793d9827b90d3f
drwx--x---  3 root root  4096 Jun 22  2022 a8a102ffcf0bb90ac0efa4d6f8f3bc435ff498622a983046f846f5fd970c360b
drwx--x---  4 root root  4096 Jun 22  2022 d5275b5d6e222fa9a4ea554515456d177529aea765267aa778d1cb442d35c4c3
drwx--x---  4 root root  4096 Jun 22  2022 3bcb7a8db99ff546cdc41e3c227387b71d616fe545cc51511cda7463a8dc83d2
drwx--x---  4 root root  4096 Jun 22  2022 d06ce7ab1a553e924bf314b1c0581e094606a2e38b5392d92f5ae614b09c70b6
drwx--x---  4 root root  4096 Jun 22  2022 d0a8f9d74eb28a8adfa0fa6c69e408298975ece602f9f264d56437c20ba424a6
drwx--x---  4 root root  4096 Jun 22  2022 6da1bde55a12d4e727e6e5f62766ab1a194eb51ae6e8c85062f4ddb3efc426f0
drwx--x---  4 root root  4096 Jun 22  2022 bfa6067837be6de3448a76d7a5ee2a582e130a2b139c483ead467ccc197e71b2
drwx--x---  4 root root  4096 Jun 22  2022 b933f451a9b42dd88992de2e96571fc6be49f36ed335ee0c80c4aead8c959b5a
drwx--x---  4 root root  4096 Jun 22  2022 c4e0dcd80fd99cef54070ae1a78d24580142162bb01ff7a6b0761a630836ef86
drwx--x---  4 root root  4096 Jun 22  2022 d8c858be1a5c74bb0bc66914417024e0f0d415615ce6799d6d29f1f7b8e97ebc
drwx--x---  4 root root  4096 Jun 22  2022 f9afce13116be85c48ddb6a01d3311a6bdad15cf894216cc5bb2c52cfe13a198
drwx--x---  4 root root  4096 Jun 22  2022 98181bbfaaf7ed8ecb78a35c965b933f76c350bc3f8066f0d05791b820fada73
drwx--x---  4 root root  4096 Jun 22  2022 2afeeb1f76e2b6381982123b072023f3580d5060de0f0118fe25285681d24817
drwx--x---  4 root root  4096 Jun 22  2022 79f3a7b8a6f2f664ff865892f49d3545ca2d871407035fd3f9f8753ae9bf5c1a
drwx--x---  4 root root  4096 Jun 22  2022 95587e4d7a1ab1c11aa59185b4bd16916e284bd7863d7d48cf62824565cf60b3
drwx--x---  4 root root  4096 Jun 22  2022 4b6b5b83a43364c0c8eb6d7eab8f67e714f30bde3ee734efe738020b68c42cf1
drwx--x---  4 root root  4096 Jun 22  2022 41f4c04fa776c96be9c66de6b18d4123376abe0e1c740eb3c8d7c2ada9199b24
drwx--x---  4 root root  4096 Jun 22  2022 51436ee2498a7b663c595a790b7c1b2f315e738f6e782867140a3a2b64b24b3b
drwx--x---  4 root root  4096 Jun 22  2022 38857dc80185f6fb461c2dd868ddc6ad61c2d9e9088aba825db6c598827e7d5d
drwx--x---  4 root root  4096 Jun 22  2022 904cb66d4a6dba473cc7bd5a19640c896627efebc42dd2f4c032c082b652c192
drwx--x---  4 root root  4096 Jun 22  2022 7dcd7eb253d365241f806bfa60656ea20f196d2b5dcfbb8a83a5acdded5187fa
drwx--x---  4 root root  4096 Jun 22  2022 526eb161800f08742a160c7ee6a5075f7257465234e57747e7f8f6165daf12fd
drwx--x---  4 root root  4096 Jun 22  2022 cec8d001007bae5abf0267dca930bc56a5255f9b929920723363f7da225cdc37
drwx--x---  3 root root  4096 Jun 22  2022 f2a822bb868b58e55ccdeb27841d967402294045df042331245f826a1d2b2ede
drwx--x---  4 root root  4096 Jun 22  2022 1872ff6d41173ea508d42baff5d770f2c6202d9bffe54baa1a4a0a87ec39be6d
drwx--x---  4 root root  4096 Jun 22  2022 da70549d6a524f27cc46f2c6166f9df522bec92355a07cf6ca5f153092b5b47a
drwx--x---  4 root root  4096 Jun 22  2022 8fd3f26b42df7b886ad7b70273f6d4d9975f11d04474b0cc36d640030095ca70
drwx--x---  4 root root  4096 Jun 22  2022 31acea78538530bdc960e029518445fe497623d49a29a637009346af9012a6e8
drwx--x---  4 root root  4096 Jun 22  2022 a120019c1f22b4347e5981300a201ae77b78208a6ef0216438ca35ae2190ebc8
drwx--x---  3 root root  4096 Jun 22  2022 9e6d51dd46218bfa6dc8adfff778caf4bb8ca806b3e68b1fe383738ba712009f
drwx--x---  4 root root  4096 May 23  2022 154fa7c01c69466d2a10d179c9808d4090700ce619921654f3c0a48f501e1bb0
drwx--x---  4 root root  4096 May 11  2022 b6fba3b66f40a612ede4f630ef482c7291c529cf19bf384193bc6ba6204f3638
drwx--x---  4 root root  4096 May 11  2022 7a1f5e1934b926893acb0171d4fbe1cfea78c69b5d293c51bc3eccba18124a22
drwx--x---  4 root root  4096 May 11  2022 0ddcc536dcc7c6c3a8e3c44b8eabd0d09d8b0c113b8d7587266cc5e542e816b1
drwx--x---  4 root root  4096 May 11  2022 2b4f4394d793628ef40ef1e8c725814fcf82677ad8e04154e2514af610bdfc97
drwx--x---  4 root root  4096 May 11  2022 19b72deef0c01b3da7328c5145b14fa19567873d57a267706c92f249727c461d
drwx--x---  4 root root  4096 May 11  2022 93a70a6f157b46e65dc50264a066ea3ae3b3fb201f19fd2bd3cd07e5a153f8e8
drwx--x---  4 root root  4096 May 11  2022 ea5fb6ea8a9ef1ac9fda608402a24b92f4249f242bf03d58adce449734170015
drwx--x---  3 root root  4096 May 11  2022 770119307f542171f209354ef27333d51901b30e8f5573521afdd9150366e09a
drwx--x---  4 root root  4096 May 11  2022 8f0021e8770633692eae358d5e3acabcb62df43a893244708ddfc38ca435dcb9
drwx--x---  4 root root  4096 May 11  2022 bdad8329b4c67e87a4491d332dcbd3deb33aac9e5fb84f1986b1caa7fdd9a5c9
drwx--x---  4 root root  4096 May 11  2022 7680040f7c57bf1774f7343fdef191494795320b3cbcc8e2b5c7cae2a183a9e6
drwx--x---  4 root root  4096 May 11  2022 3a9817e4637a3f3dcf9cdcdace4c891372e4f4d0e9b53757d33086a3fb3f123c
drwx--x---  4 root root  4096 May 11  2022 1d4ddbe9c60964664cee7752af536c8ecfde82b24e540d788124c401a484660d
drwx--x---  3 root root  4096 May 11  2022 3226be1fbacd13d59cfc71fc7756f151f1e3e6965622ba04528e4f92998a45f7
#
# du -sh /var/lib/docker/overlay2
11G	/var/lib/docker/overlay2
```

**Comment by gustavotrott** [comment]
I think we need another way to clear this directory.

It seems that `docker system prune` removes unused images.
![image](https://user-images.githubusercontent.com/5660191/216969667-60e51a12-3e39-44b8-aedd-03904e666ef2.png)


It turned out that the Libreoffice container is destroyed immediately after finishing the conversion.
So `docker system prune` considers this image as unused.

![image](https://user-images.githubusercontent.com/5660191/216969057-89bf7403-5b1e-47ab-a4b8-c8544e1f2750.png)

In short: `docker system prune` removes the Libreoffice image and breaks the conversion script used by BBB.

**Comment by GuiLeme** [comment]
So, do we want to keep a history of say 5 days, or can we simply remove them all periodically? 

 - First case: every day deletes folders older than 5 days;
 - Second case: every say 5 days we delete all folders, regardless of their creation date.

**Comment by GuiLeme** [comment]
Hey, @ffdixon, could you give us more information on that folder and the docker?

Such as: the output of `du -sh /var/lib/docker/overlay2/* | sort -hr`, `docker image ls`, and any other command that you judge it would help us.
